src/Home/Home.tsx

```diff
@@ -0,0 +1,274 @@
+import { Box, Container, Grid, Typography } from "@mui/material";
+import { Card } from "@mui/material";
+import { useContext, useEffect } from "react";
+import BedIcon from "@mui/icons-material/Bed";
+import AdsClickIcon from "@mui/icons-material/AdsClick";
+import WorkOutlineIcon from "@mui/icons-material/WorkOutline";
+import {
+  ResponsiveContainer,
+  PieChart,
+  Pie,
+  Cell,
+  Legend,
+  Tooltip,
+} from "recharts";
+import { contextDashBoard } from "../DashBoardRoomContext/DashBoardRoom";
+
+export default function Home() {
+ 
+
+  const {
+    DataDashboard,
+    numberForFacilities,
+    numberForRooms,
+    numberForAds,
+    userData,
+    AdminData,
+    pendingBookings,
+    completedBookings
+  } = useContext(contextDashBoard);
+
+  const Colors1 = [
```

Comment:
We will need to have different set of colors for the dark theme as well, or if we use the MUI charts and used the colors in the theme, they will be managed by MUI.

Reply:
We need to look here for styling charts under the provided pallet:
https://mui.com/x/react-charts/styling/